barretenberg/cpp/src/barretenberg/ultra_honk/protogalaxy.test.cpp

```diff
@@ -1,345 +1,412 @@
+#include "barretenberg/goblin/mock_circuits.hpp"
```

Comment:
I haven't really modified these tests except from typing and differentiating between the type of circuit that needs to be constructed in `construct_circuit`

Reply:
Ahh but since you've modified some stuff in this file it makes it all fair game :). My usual comment about lack of documentation applies to this file. It's ok with me if you don't want to do that in this PR but if you don't we should make an issue to come back to it soon. I totally understand how this slipped through without comments since it was primarily you and Cody in a bubble working on this stuff but as I go to start integrating PG, the tests will be my guide. It doesn't need to be crazy, but at the very least each method should have a high level description of what it intends to test.